test/api/core/entrypoints/embedded/protocols/mqtt.test.js

```diff
@@ -58,48 +58,59 @@ describe('/lib/api/core/entrypoints/embedded/protocols/mqtt', () => {
     };
 
     protocol = new MqttProtocol();
-    return protocol.init(entrypoint);
   });
 
-  describe('#init', () => {
+  describe('#init', () => {    
+    it('should return false if the protocol is disabled', () => {
+      entrypoint.config.protocols.mqtt.enabled = false;
+
+      return protocol.init(entrypoint)
```

Comment:
(nitpicking) can be simplified with [fulfilledWith](https://shouldjs.github.io/#assertion-fulfilledwith):

```js
return should(protocol.init(entrypoint)).fulfilledWith(false);
```